### PR TITLE
ES: Ep2 Un

### PR DESCRIPTION
**Pull request by NozomiRem**

### CODE REVIEW

**Comment by TheBlazter** [comment]
**Found duplicate files**
un_020160.csv

File: Story/Episode 4/st_042130.csv [hunk 1]
```diff
@@ -0,0 +1,16 @@
+st_042130_0001#0,"""A.D.2028/3/30/12:30"""
+st_042130_0010#0,"""...¿Eh? ¿Esta es mi habitación? ¿Regresé?... ¿Q-qué ?! ¡¿Cómo?!"""
+st_042130_0020#0,"""...¿¡Una desconexión forzada de la infraestructura Ether !? Código de error: \""Olympia\""... Que es eso..."""
+st_042130_0030#0,"""The Ether Infrastructure around here has been cut off? Can't even use mobile phone... What's happening here!?"""
```
Comment (alama) [inline Story/Episode 4/st_042130.csv]: `st_042130_0030#0` is still in EN

Comment (TheBlazter) [reply Story/Episode 4/st_042130.csv]: i read the lines, and is partial translated on Spanish, and still part on English

Comment (TheBlazter) [reply Story/Episode 4/st_042130.csv]: Please wait i translate it

**Pull request merged**